src/Tizen.Nlp/Interop/Interop.Nlp.cs

```diff
@@ -0,0 +1,306 @@
+/*
+ * Generated by tidlc 1.2.4.
+ *
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Tizen.Applications;
+using Tizen.Applications.RPCPort;
+namespace RpcPort
+{
+namespace Message
+{
+    namespace Proxy
+    {
+        internal class message : ProxyBase
```

Comment:
Are you sure the message class is internal? Why did you put the internal class in the design document?

Reply:
i used csuml tools to generate the class diagram , the link is https://github.sec.samsung.net/p-szydelko/csuml , 
when i change  the proxy class from public to internal ,the tools still generate the class diagram with it ? so ,
how can i remote it ?